tests/rpmi.at

```diff
@@ -1371,3 +1371,21 @@ runroot rpm -q --whatprovides /
 ],
 [])
 AT_CLEANUP
+
+AT_SETUP([rpm -U with Obsoletes])
+AT_KEYWORDS([install])
+AT_CHECK([
+RPMDB_INIT
+
+runroot rpmbuild -bb --quiet /data/SPECS/test-obsoletes.spec
+runroot rpm -i --nodeps --ignorearch /data/RPMS/hello-1.0-1.i386.rpm
+runroot rpm -U /build/RPMS/noarch/test-obsoletes-1.0-1.noarch.rpm
+runroot rpm -q hello
+runroot rpm -q test-obsoletes
```

Comment:
Just run these two in a single command, that way you get the combined return code of 1.

Reply:
Thank you for your comments. I have made relevant changes. Please review the changes.